react-charts/area-chart/step1.md

```diff
@@ -1,10 +1,22 @@
-PatternFly React charts are based on the [Victory chart](https://formidable.com/open-source/victory/docs/victory-chart/) library, along with additional functionality, custom components, and theming for PatternFly. 
-This provides a collection of React based components you can use to build PatternFly patterns with consistent markup, styling, and behavior. 
+This step will begin by adding a CSS selector to define the chart size.
 
-PatternFly React charts are composed using separate components, so features are more easily customizable. 
-In this course, we're going to build a PatternFly area chart together - starting with a simple chart, adding multiple datasets, tooltips, axis labels, a legend, and concluding by changing the theme color.
-You'll learn how to use PatternFly React chart components together to build a consistent user experience.
+Note: Katacoda is setting up a new React application for you. You will be ready to code once the server starts and you can see "Welcome to PatternFly" on the lower pane.
 
-The image below is what we'll end up with when we are done.
+1) <strong>Navigate to the `src` folder and open `src/app.css`{{open}}</strong>

```

Comment:
The file doesn't exist by default, so wording should lead to create it first.

Reply:
It does. The file is created via the env-init.sh file.

`touch ./src/app.css`

Reply:
We can move that command at the first line. So the file would be there before `npm install` which takes a long time.
I'm wondering if we couldn't put `node_modules` already in the `pf-react-express.tar.gz` and skip `npm install` at all. I think that it doesn't need to do that every time. It takes almost minute before the project is running.

Reply:
First two lines would be:
`mkdir -p ~/tutorial/src
touch ~/tutorial/src/app.css`

Reply:
Thoughts @dlabrecq

Reply:
Adding the charts and tokens packages is possible, but pf-react-express.tar.gz would need to manually updated regularly -- don't know when we updated that last?

I'll defer to @dgutride as to what we want to do here. Perhaps we can automate the build for pf-react-express.tar.gz?

Reply:
Lets make this an issue we look at separately. We're currently investigating how best to use a Docker image.

Reply:
If we move the `touch ./src/app.css` command, it will still need to come after we un-tar `pf-express.tar.gz`. That is, not the very first two lines.